### ISSUE TITLE
TRPC community.get members returns 500 error

### ISSUE THREAD

**Issue by ilijabojanovic** [hicommonwealth/commonwealth#7321]
## Describe the bug

Members page returning 500 error on api call

### Initial conditions

**Environment:** 

QA env / Local
**Branch/Release version:** 

release/v1.2.4-x

**Browser:** 

All
**Wallet:** 

All

## Reproduction steps

1. Go to https://qa.commonwealth.im/ray-demo/members?tab=all-members
2. Check console

## Actual behavior

```
[{error: {message: "Stakeholder group not found", code: -32603,…}}]
0
: 
{error: {message: "Stakeholder group not found", code: -32603,…}}
error
: 
{message: "Stakeholder group not found", code: -32603,…}
```

## Expected behavior

We should not return 500 error

## Screenshots / Video

https://app.screencast.com/1ZXceb5vaDs0k

**Comment by Rotorsoft** [comment]
@rbennettcw We shouldn't return invalid state errors from queries... by definition, a query just gets what's being projected (already happened)... invalid state if when commands cannot be executed b/c it violates an invariant.

```
      if (!stakeholderGroup) {
        throw new InvalidState(Errors.StakeholderGroup);
      }
      const node = await models.ChainNode.findByPk(community.chain_node_id);
      if (!node) {
        throw new InvalidState(Errors.ChainNodeNotFound);
      }
```

in this case, we have to decide if these results are hidden or flagged in the results of the query

**Comment by rbennettcw** [comment]
Just realized this is QA, not production.

➡️ The immediate solution is to run the generate script in QA and confirm if it creates groups as expected.

There's a chance that the QA DB was blown away, which removed the generated groups. The generate script needs to be run manually to create the groups (since it makes onchain calls to get the namespace address which is used for rules, it can't be a migration).

➡️ For a better long-term solution, the script should be a migration. Tim recently started working on copying namespace addresses to the DB, which means that the generate script will no longer require network calls and it can be a migration.

**Comment by Rotorsoft** [comment]
Immediate solution sounds good! Let's make sure we address the following in the near future:
- Agree on a query pattern we want to establish and validate when reviewing PRs -> (like not raising errors as discussed, or any other convention we find necessary)
- How to report data errors violating business rules? Testing, rollbar, etc
- Agreed to use migrations to fix data errors (manual scripts might be forgotten)

**Comment by rbennettcw** [comment]
Root cause was that the stakeholder script didn't properly select which communities to create groups for.

Fixed in #7337.
